reactors/1D_packbed.ipynb

```diff
@@ -0,0 +1,518 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# One-dimensional packed-bed, catalytic-membrane reactor model without streamwise diffusion"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The present model simulates heterogeneous catalytic processes inside packed-bed, catalytic membrane reactors. The gas-phase and surface-phase species conservation equations are derived and the system of differential-algebraic equations (DAE) is solved using the scikits.odes.dae IDA solver."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Runnning Cantera version: 2.5.1\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Import Cantera and scikits\n",
+    "import numpy as np\n",
+    "from scikits.odes import dae\n",
+    "import cantera as ct\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "%matplotlib inline\n",
+    "print(\"Runnning Cantera version: \" + ct.__version__)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Methodology\n",
+    "\n",
+    "One-dimensional, steady-state catalytic-membrane reactor model with surface chemistry is developed to analyze species profiles along the length of a packed-bed, catalytic membrane reactor. The same model can further be simplified to simulate a simple packed-bed reactor by excluding the membrane. The example here demonstrates the one-dimensional reactor model explained by G. Kogekar, et al., \"Efficient and robust computational models of  heterogeneous catalysis in catalytic membrane reactors,\" (2022), In preparation."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Governing equations\n",
+    "\n",
+    "Assuming steady-state, one-dimensional flow within the packed bed, total-mass, species mass and energy conservation may be stated as [1]\n",
+    "\n",
+    "$$\n",
+    "    \\frac{{\\rm d}(\\rho u)}{{\\rm d}z} = \\sum_{k=1}^{K_{\\text{g}}} \\dot s_k W_k A_{\\text{s}} + \\frac{P_{\\rm b}}{A_{\\rm b}}       j_{k_{\\rm M}}, \\\\\n",
```

Comment:
```suggestion
    "    \\frac{d(\\rho u)}{dz} = \\sum_{k=1}^{K_{\\text{g}}} \\dot s_k W_k A_{\\text{s}} + \\frac{P_{\\mathrm{b}}{A_{\\mathrm{b}}       j_{k_{\\mathrm{M}}, \\\\\n",
```
and similar elsewhere. Please also consistently use `\\text` and `\\mathrm`

Reply:
Corrected.